### ISSUE TITLE
Go to definition opens components.d.ts and desired component

### ISSUE THREAD

**Issue by tilenpirih** [antfu/vscode-goto-alias#19]
### Describe the bug

When I ctrl + click on it opens components.d.ts and a component that I want to open. It is just an annoying. Is there any hack to solve this or do I need to change some vscode settings?

https://github.com/user-attachments/assets/745f0a67-b5ed-4a61-8ff5-3694df693041

### Reproduction

Opens also components.d.ts

### System Info

```Shell
System:
    OS: Windows 11 10.0.22631
    CPU: (24) x64 AMD Ryzen 9 7900X3D 12-Core Processor
    Memory: 40.80 GB / 63.16 GB
  Binaries:
    Node: 20.10.0 - C:\Program Files\nodejs\node.EXE
    Yarn: 1.22.22 - C:\Program Files\nodejs\yarn.CMD
    npm: 10.2.3 - C:\Program Files\nodejs\npm.CMD
    pnpm: 8.14.1 - ~\AppData\Local\pnpm\pnpm.EXE
    bun: 1.1.21 - ~\.bun\bin\bun.EXE
  Browsers:
    Edge: Chromium (127.0.2651.74)
    Internet Explorer: 11.0.22621.3527
```

### Used Package Manager

bun

### Validations

- [X] Follow our [Code of Conduct](https://github.com/antfu/.github/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/antfu/contribute).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

### Contributions

- [ ] I am willing to submit a PR to fix this issue
- [ ] I am willing to submit a PR with failing tests (actually just go ahead and do it, thanks!)

**Comment by Ricro** [comment]
Actually, I just found out in the source code that there is an option for this. To enable it: 

1. CMD + SHIFT + P > Open User Settings
2. Look for "goto alias"
3. Tick "Close Dts" checkbox

![CleanShot 2024-10-16 at 11 00 14@2x](https://github.com/user-attachments/assets/e9b94bb0-0a54-41b2-873b-70a268a88164)

It would be nice to enable it by default, or at mention it in the README.md

**Comment by tilenpirih** [comment]
> Actually, I just found out in the source code there is an option for this. To enable it:
> 
> 1. CMD + SHIFT + P > Open User Settings
> 2. Look for "goto alias"
> 3. Tick "Close Dts" checkbox
> 
> ![CleanShot 2024-10-16 at 11 00 14@2x](https://private-user-images.githubusercontent.com/16501699/376973935-e9b94bb0-0a54-41b2-873b-70a268a88164.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzAxMTA5OTMsIm5iZiI6MTczMDExMDY5MywicGF0aCI6Ii8xNjUwMTY5OS8zNzY5NzM5MzUtZTliOTRiYjAtMGE1NC00MWIyLTg3M2ItNzBhMjY4YTg4MTY0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDEwMjglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQxMDI4VDEwMTgxM1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkxYzQwODYxOWMyYmQ3NDA5Y2JlMGUxMWJiN2EwOTg2YWE1OGJkYmM2N2MxYTBmOGEwNmEwZTllMWRmNTQ4NDAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.idKm9BeGUF6xy9dnzGV9bmF1yxqEDKUZ18tj6LqDR98)
> 
> It would be nice to enable it by default, or at least mention it in the README.md

I agree... I made a [PR ](https://github.com/antfu/vscode-goto-alias/pull/22)

**Comment by husayt** [comment]
For me with Nuxt it just take you to d.ts file and doesn't even open the target. Not sure what am I missing

**Comment by tilenpirih** [comment]
> For me with Nuxt it just take you to d.ts file and doesn't even open the target. Not sure what am I missing
> 

If I'm not mistaken nuxt4 is also smarter with this and it fixes this issue. Will do a test
